.github/workflows/arviz_compat.yml

```diff
@@ -1,7 +1,7 @@
 name: arviz-compatibility
 
 on:
-  pull_request:
+  # pull_request:
```

Comment:
TODO: revert

Reply:
> The links in the In-Depth Guides section are broken, but I guess that's known?

Fixing it now.

> If we merge his, it won't affect http://docs.pymc.io until we redirect the DNS registration, right?

Yes. After merging, the v3 docs will be rendered here https://pymc3.readthedocs.io/en/v3/

> Also it will need to be cherry-picked onto main, right?

I want to play around doing similar things for the main branch. It is one option, yes. But I am not sure 100% sure if this will work.

Reply:
The new docs https://pymc3-fork.readthedocs.io/en/v3_rtd_docs/ are built on latest commit 7569d8c and it fixes the broken link to in-depth guides.